Bricks/README.md

```diff
@@ -15,18 +15,13 @@ HTTP server and client, JSON and binary serialization, visualization and other c
   [tar.gz](https://github.com/C5T/Current/archive/v1.0.tar.gz),
   [zip](https://github.com/C5T/Current/archive/v1.0.zip).
 
-  HTTP server and client API-s, RapidJSON for JSON serialization, [gnuplot](http://www.gnuplot.info/) and [plotutils](http://www.gnu.org/software/plotutils/) ports, file system and string manipulation methods, a command line flags parsing tool and a header-only port of [GoogleTest](http://code.google.com/p/googletest/).
+  HTTP server and client API-s, [Cereal](http://uscilab.github.io/cereal/) for binary and JSON serialization, [gnuplot](http://www.gnuplot.info/) and [plotutils](http://www.gnu.org/software/plotutils/) ports, file system and string manipulation methods, a command line flags parsing tool and a header-only port of [GoogleTest](http://code.google.com/p/googletest/).
```

Comment:
I went on to fix it too upon noticing, only to realize these are the release notes of Bricks 1.0. :-) Suggest we keep it for now, and retire the whole section once we release Current 1.0.